### DISCUSSION TITLE
Welcome to web3 Discussions!

### DISCUSSION THREAD

**Discussion by rufuspollock** [rufuspollock/awesome-crypto-critique#54] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:

* Ask questions you’re wondering about.
* Share ideas.
* Engage with other community members.
* Welcome others and are open-minded. Remember that this is a community we
build together 💪.

To get started, introduce yourself in the introductions section https://github.com/life-itself/web3/discussions/categories/introductions tell us about yourself, your skills and how you'd like to contribute to the community.

**Comment by georgepor** [comment]
Thank you, @rufuspollock for starting this!

As I commented on the Emerge tweet on your initiative:

"This is a timely response to the Zeitgeist.
Many of us who continue our sense-making abt Web3 & DAOs in a "learning-by-doing" mode, will gather in [The Rendezvous](https://futurehow.site/else/the-rendezvous/) of March 20-21. 

<img width="997" alt="The Rendezvous with text" src="https://user-images.githubusercontent.com/73591111/158619670-285d88ee-8a1b-4721-bfcf-cfdc51bed2bd.png">

If you want to grab one of the few remaining seats, head over to [Registration](https://docs.google.com/forms/d/e/1FAIpQLSdrTOOvqMLi2_0A9NX0oTJvQcSKbrXoi0wCfLEG8ljcZAL4Yw/viewform)."

The Rendezvous is not a one-off event, only the first milestone of a multi-year **Generative Action Research on _Enlivening Value Flows_** in and across the ecosystems of social movements contributing to [humanity's Phase Shift](https://www.youtube.com/watch?v=nQRzxEobWco), supported by Web3 and social impact DAOs.

All DAOs are not equal! The social impacts DAOs are  a universe to itself that has:
[All for Climate DAO ](https://twitter.com/all4climatedao)
[BiodiversityDAO](https://twitter.com/BiodiversityDAO)
[Blockchain for Humanity (b4H)](https://twitter.com/b4_humanity)
[CityDAO](https://twitter.com/CityDAO)
[Crypto Commons Association](http://linktr.ee/CCommonsHub)
[Dream DAO](https://cuwhitepapers.notion.site/88f1e7564e9d44fd86ece3bfbc5716e5?v=d8e282647c284625b226039f0e7589c8) 
[Popcorn DAO](https://popcorn.network/)
[Regen Foundation](http://regen.foundation/)
[ Regenerative Economies  SEEDS](https://twitter.com/SEEDSeconomy)
[Sovereign Nature Initiative](https://twitter.com/sovereignnature)
[The Commons Stack](http://commonsstack.org/apply)
just to name a few... 

Disclosure: I am advising and consulting to a number of them.

If you guys want to get into the _action_, feel free to reach out.

**Comment by jessirish** [comment]
Thank you so much for starting this. Here is the question I keep coming back to:
- Are there any active proponents of web3 who are not "bought in" to the financial aspects of cryptocurrencies, bubble markets, etc? From my POV it's hard to try and listen to the "pro" side if it all feels like shilling and boosterism.

**Reply by georgepor** [reply]
> hard to try and listen to the "pro" side if it all feels like shilling and boosterism.

I agree. @jessirish, why not try some of these:

- [On the Potential of DAOs & Web3 with Jordan Hall & Rufus](https://www.youtube.com/watch?v=aQvpgYW0o7M)
- [The Blockchain Revolution by Michel Bauwen](https://greattransition.org/gti-forum/tech-future-bauwens)
- [Web3 relies on participatory economics, and that is what is missing — Participation](https://cointelegraph.com/news/web3-relies-on-participatory-economics-and-that-is-what-is-missing-participation)
- [What ​A​re Socially Owned Social Platforms and How Will They Solve the Problems of Social Networks?](https://anatha.io/blog/socially-owned-social)
- [web3 allows us to program our values into our money](https://twitter.com/owocki/status/1504138392837058564)
- [DAO’s don’t work](https://blog.nestr.io/daos-don-t-work-5dfcb7d49ebb)

That is just from the top of my head...

**Reply by jessirish** [reply]
great, I'll check these out.

**Reply by georgepor** [reply]
> great, I'll check these out.

 I’d be curious to learn what you think of them.

**Reply by georgepor** [reply]
![Screenshot 2022-03-16 at 18 07 25](https://user-images.githubusercontent.com/73591111/158658055-353aff46-1f1e-4dde-a8fc-be817fdfd948.png)

"This book is about cryptoeconomic systems that create positive externalities for their neighbors and for the world.

We explore the intersection of programmable money, game theory, and mechanism design. We search for powerful new ways to fund, design, develop, and market regenerative web3 era applications and digital assets. We launch the meme of regenerative cryptoeconomics into the world."  — https://www.blurb.co.uk/b/11078160

**Reply by robo360** [reply]
Thanks @georgepor for the resources.

**Reply by georgepor** [reply]
> Thanks @georgepor for the resources.

They are a very small fraction of my research library of our _action_-research about the use of Web3 and social impact DAOs, which, presently resides only on my hard disk. It is waiting for a web developer to help implement its knowledge architecture in a GitBook-based Community Knowledge Garden. (Not easy to find one because it's an unfunded project.)

**Reply by rufuspollock** [reply]
@georgepor first welcome and thanks for joining and sharing.

@georgepor i've interacted a little with the author of the green pilled book - see e.g. this thread https://github.com/life-itself/web3/discussions/72 I'd be interested to know what stuff is in the book - is. there a summary or outline somewhere?

You'd be welcome to add material to the collective zotero library perhaps starting in your own personal section under `~/` collection.
